.appveyor.yml

```diff
@@ -9,7 +9,7 @@ platform:
 - x86
 environment:
   matrix:
-  - PYTHON: 36
+  - PYTHON: 38
```

Comment:
> I was forced to set up a newer version of python (3.8) in CI.

This sounds like a misunderstanding. Whatever is done in this PR should not break the existing test (because it would also break users). Could you please undo this change?

(I think dropping Python 3.6 support is past due, but that's a separate step.)